doc/md_src_files/040_stakeholders-scope.md

```diff
@@ -114,9 +114,14 @@ Table 2: Proposed Meaning for Supplier Priority Outcomes
 
-When remediation is available, usually the action is to apply it. When remediation is not yet available, the action space is more diverse, but it should involve mitigating the vulnerability (e.g., shutting down services or applying additional security controls) or accepting the risk of not mitigating the vulnerability. Table 3 displays the action priorities for the deployer, which are similar to the supplier case.
+When remediation is available, usually the action is to apply it. When remediation is not yet available, the action space is more diverse, but it should involve mitigating the vulnerability (e.g., shutting down services or applying additional security controls) or accepting the risk of not mitigating the vulnerability. Applying mitigations may change the value of decision points. For example, effective firewall and IDS rules may change [*System Exposure*](#system-exposure) from open to controlled. Financial well-being impact might be reduced with adequate fraud detection and insurance. Physical well-being impact might be reduced by physical barriers that restrict a robot's ability to interact with humans. Mission impact might be reduced by introducing back-up business flows that do not use the vulnerable component.
+
+Mitigation techniques may not affect the SSVC decision. For example, the implementation of a firewall or IDS rule to mitigate [*System Exposure*](#system-exposure) from open to controlled is only valid until someone changes the rule. In the area of Financial impacts, the caps on the insurance may be too low to act as a mitigation. The Physical impact may be increased by incorrect installation of the physical barriers designed to restrict a robot’s ability to interact with humans. The Mission impact could be increased when a disaster recovery test-run identifies problems with an alternate business flow. 
+
+A mitigation that successfully changes the value of a decision point may shift the priority of further action to a reduced state. Implementing an adequate change control process for maintaining firewall or IDS rules may be enough to reduce the priority where no further action is necessary. In the area of Financial impacts, a better insurance policy may be purchased, providing necessary fraud insurance. Physicial well-being impact may be reduced by testing the physicial barriers designed to restrict a robot's ability to interact with humans. Mission impact could be reduced by correcting the problems identified in a disaster recover test-run of the alternate business flow. If applying a mitigation reduces the priority to **defer**, the deployer may not need to apply a remediation, later if it becomes available. Table 3 displays the action priorities for the deployer, which are similar to the supplier case.
```

Comment:
> Implementing an adequate change control process for maintaining firewall or IDS rules 

Suggest instead:
An effective firewall or IDS rule coupled with an adequate change control process for rules

Reply:
81d2fd6